doc/jobs/prerelease_jobs.rst

```diff
@@ -115,13 +115,6 @@ scripts which can be invoked manually in the same order:
   ``colcon test-result --all`` to output a summary of all tests in the overlay
   workspace.
 
-Run the *prerelease* job on Travis
```

Comment:
I'm not sure if suggesting pre-release jobs is as worthy of preserving. I know they work in theory but I think test suite quality makes these pretty difficult to parse.

Reply:
I left this one out, but let me know if you change your mind and I can easily do the same thing here I did for the above ones.